docs/proposals/35-observability2.0-victoria-logs.md

```diff
@@ -0,0 +1,231 @@
+metadata:
+  name: example
+  namespace: shoot--local--local
+spec:
+  storage:
+    resources:
+      requests:
+        storage: 30Gi
+  resources:
+    requests:
+      memory: 100Mi
+      cpu: 10m
+    limits:
+      memory: 300Mi
+      cpu: 50m
+```
+
+#### Sending Logs to VictoriaLogs in the `Shoot` control plane
+
+Currently, `Vali` access is behind a service in the namespace of the `Shoot` control plane.
+For external access, there exists an ingress in front of the `OpenTelemetry Collector` in the control-plane of `Shoot` clusters that is used to post logs to `Vali`.
+See [GEP-34](./34-observability2.0-opentelemtry-operator-and-collectors.md) for more details.
+
+The same setup will continue with `VictoriaLogs`, with the only difference being that the `OpenTelemetry Collector` will be configured to ship logs to `VictoriaLogs` via `OTLP` instead of via the `Loki` protocol.
+
+### 4. Migration from Vali to VictoriaLogs
+
+#### Data Migration
+During all the replacements of `Vali` we want to ensure that existing clusters do not get disrupted.
+For this reason, 2 strategies for migration have been considered:
+- "One-shot" migration.
+Reformat all the Vali data into the VictoriaLogs format (or OTLP).
+Could be done via the OpenTelemetry Collector since there is no tool that exists to do this out of the box.
+
+    Pros:
+    - Migration step is quick.
+    - No friction during the migration period since there is only one source of logs after the migration.
+
+    Cons:
+    - Development of the necessary tooling to do the migration.
+    - Risk of data loss if something goes wrong during the migration.
+
+- "Dual" migration.
+`VictoriaLogs` get deployed alongside `Vali` and *both* backends receive logs.
+
+    Pros:
+    - No risk of data loss.
+    - No friction during the migration period since all logs are visible in both backends.
+
+    Cons:
+    - Higher storage cost due to the replication of the logs on both backends.
+
+Due to the "dual" migration not requiring the development of new tooling and having a lower risk of data loss, it has been chosen as the migration strategy.
+Both `Garden` clusters and `Seed` clusters will rely on this strategy, as well as new and existing `Shoot` clusters.
+The deployment alongside `Vali` will be controlled using two feature gates.
+
+##### Dual Migration Implementation Details
+
+To facilitate the "dual" migration, a feature gate `VictoriaLogs` will be introduced.
+When this feature gate is enabled:
+- New  and existing clusters will have `VictoriaLogs` deployed *alongside* `Vali`.
+- The OpenTelemetry Collector will be reconfigured to ship logs to *both* `Vali` and `VictoriaLogs`.
+- `FluentBit` instances will be reconfigured to ship logs to *both* `Vali` and `VictoriaLogs`.
+
+##### Removing Vali after the migration period
+
+Removal of Vali will further rely on another feature gate called `SkipVali` that will rely on the `VictoriaLogs` already being enabled for the specific cluster.
```

Comment:
Can you document what the behaviour will be if `SkipVali` is enabled, but `VictoriaLogs` is not enabled? Is the plan to forbid this configuration by causing a panic? 

I vote to name the feature gates `RemoveValiAfterLogRetentionPeriod` and `DeployVictoriaLogs`